### PR TITLE
Add `skip_after` option to flamegraph

### LINKED ISSUE #171: allow customization of CollapseOptions
inferno's `CollapseOptions` have some useful tunings. It could be nice if we could customize those via flamegraph. 
See https://github.com/flamegraph-rs/flamegraph/blob/main/src/lib.rs#L271  where they are instantiated.

1. Would you agree to this enhancement?
2. How would you suggest to propagate them? One way is to explicitly declare each option and manually propagate them, 
but I thought it could be nice to do something like `--collapse_options=key1=val1,key2=val2...` and let the underlying struct deserialize those key pairs. This approach would reduce the maintenance overhead and give 100% coverage of these options.

### PR DESCRIPTION
**Pull request by romange**
Fixes #171

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -108,6 +108,15 @@ mod arch {
             .context("unable to call perf script")?
             .stdout)
     }
+
+    pub fn set_collapse_options(
```
Comment (djc) [inline src/lib.rs:112]: I'd probably prefer not to have this as a separate method -- I think we could just inline this code?

Comment (romange) [reply src/lib.rs:112]: it should be under cfg attribute conditional. How can I inline a milti-line block under `#cfg` attribute?

Comment (djc) [reply src/lib.rs:112]: `#[cfg()] { /* your code goes here */ }`

Comment (romange) [reply src/lib.rs]: fixed

**Review by djc — APPROVED** [review]

**Comment by djc** [comment]
Thanks!

**Pull request merged**